docs/content/middlewares/stripprefix.md

```diff
@@ -69,32 +69,55 @@ http:
 
 ### General
 
-The StripPrefix middleware will:
-
-- strip the matching path prefix.
-- store the matching path prefix in a `X-Forwarded-Prefix` header.
+The StripPrefix middleware strips the matching path prefix and stores it in a `X-Forwarded-Prefix` header.
 
 !!! tip
-    
-    Use a `StripPrefix` middleware if your backend listens on the root path (`/`) but should be routeable on a specific prefix.
+
+    Use a `StripPrefix` middleware if your backend listens on the root path (`/`) but should be exposed on a specific prefix.
 
 ### `prefixes`
 
 The `prefixes` option defines the prefixes to strip from the request URL.
 
-For instance, `/products` would match `/products` but also `/products/shoes` and `/products/shirts`.
-
-Since the path is stripped prior to forwarding, your backend is expected to listen on `/`.
-
-If your backend is serving assets (e.g., images or Javascript files), chances are it must return properly constructed relative URLs.  
-Continuing on the example, the backend should return `/products/shoes/image.png` (and not `/images.png` which Traefik would likely not be able to associate with the same backend).  
+For instance, `/products` also matches `/products/shoes` and `/products/shirts`.
 
-The `X-Forwarded-Prefix` header can be queried to build such URLs dynamically.
+If your backend is serving assets (e.g., images or Javascript files), it can use the `X-Forwarded-Prefix` header to properly construct relative URLs.
```

Comment:
```suggestion
If your backend is serving assets (e.g., images or JavaScript files), it can use the `X-Forwarded-Prefix` header to properly construct relative URLs.
```